docs/proposals/version_documentation.md

```diff
@@ -0,0 +1,97 @@
+---
+title: 'Building A Versioning Plugin For Thanos'
+linkTitle: 'Building A Versioning Plugin For Thanos'
+---
+
+- Author: [Uchechukwu Obasi](https://github.com/thisisobate)
+- Date: May 2020
+- Status: proposal
+
+## Problem
+
+The Thanos website contains the docs area which is built by rendering the markdown files from the tip of master of the Thanos repository. This frequently causes confusion for users, as breaking changes are often merged into master that conflicts with the APIs of previous releases.
```

Comment:
```suggestion
The Thanos website contains the docs area which is built by rendering the markdown files from the tip of master of the Thanos repository. This frequently causes confusion for users, as different Thanos versions have different features.
```

Reply:
FYI we almost never break our users or do conflicting APIs.

Reply:
Wow! So sorry for the misconception. Would commit your suggestion ASAP.
Thanks for letting me know @povilasv